models/stg_zuora.yml

```diff
@@ -74,9 +74,9 @@ models:
           - not_null
       - name: category
         description: The accounting category associated with the table.
-        tests:
-          - accepted_values:
-              values: ['Assets', 'Liabilities', 'Equity', 'Revenue', 'Expenses']
+        tests: 
+          - accepted_values: 
+              values: ['Assets', 'Liabilities', 'Equity', 'Revenue', 'Expenses'] 
```

Comment:
These types of tests have typically come back to burn us in the long run since the source can change without us being prepared for the change. Unless it is paramount that the field have **only** the accepted values, then I would request we remove these tests.

I don't want to over test and have users with production environments be presented with a number of test failures. Would you be able to remove all the accepted_values tests in this package.

Reply:
Removed.  Good point about changes in the source.

Reply:
It looks like one is still left in the tests. Can we remove it [here](https://github.com/fivetran/dbt_zuora_source/blob/dec2f5f2553787b54b548165b3e38d6f4b4bf192/models/stg_zuora.yml#L146-L148)